Druid/DRPiNoneMaskConstraint.class.st

```diff
@@ -6,6 +6,13 @@ Class {
 	#tag : 'IR-Paths'
 }
 
+{ #category : 'arithmetic' }
+DRPiNoneMaskConstraint >> asZ3ConstraintOnVar: aZ3Variable [
+
+	^ (aZ3Variable bitAnd: (self ifConstantDo: #yourself ifNot: #name))
```

Comment:
I have seen this `(self ifConstantDo: #yourself ifNot: #name)` already somewhere else. Maybe time to extract it?